services/datamanager/builtin/builtin.go

```diff
@@ -532,6 +556,15 @@ func (svc *builtIn) Reconfigure(
 			svc.closeSyncer()
 		}
 	}
+
+	if !svc.captureDisabled {
+		fileDeletionCtx, cancelFunc := context.WithCancel(context.Background())
+		svc.fileDeletionRoutineCancelFn = cancelFunc
+		svc.fileDeletionWg = &sync.WaitGroup{}
+		svc.fileDeletionWg.Add(1)
+		go pollFilesystem(fileDeletionCtx, svc.fileDeletionWg, svc.captureDir, &svc.syncer, svc.logger)
```

Comment:
Is there anyway to check for the fileDeletionWg [here](https://github.com/viamrobotics/rdk/blob/bea1150cdbd8d039a601777d0e77a94b7c073cbf/services/datamanager/builtin/sync_test.go#L695). It should be initialized unconditionally if capture is enabled, correct?

Reply:
It would be, but I don't think I'm understanding your suggestion/missing something, why would we need to check the Wg there?

Reply:
Just to make sure it's being set on reconfigure/init!

Reply:
I was missing something, I didn't realize the link you sent was in the `sync_test.go` file, turns out `builtin.go` and that file are really similar on line 695 lol. Added to the test, thank you!